bandit/src/query-lambda/build-query.ts

```diff
@@ -11,6 +11,7 @@ export const buildQuery = (
 	const endTimestamp = end.toISOString().replace("T", " ");
 	const startTimestamp = start.toISOString().replace("T", " ");
 	const dateForCurrencyConversionTable = subDays(start, 1); //This table is updated daily  but has a lag of 1 day
+	const channel = (test.channel.toLowerCase() === 'epic') ? 'ACQUISITIONS_EPIC' : (test.channel.toLowerCase()=== 'banner1') ? 'ACQUISITIONS_ENGAGEMENT_BANNER' :'ACQUISITIONS_SUBSCRIPTIONS_BANNER';
```

Comment:
Perhaps a cleaner approach here would be to define a mapping that makes the possible values a bit clearer. E.g.
```
type Channel = 'Epic' | 'Banner1' | 'Banner2';
const ComponentTypeMapping: Record<Channel,string> = {
	'Epic': 'ACQUISITIONS_EPIC',
	'Banner1': 'ACQUISITIONS_ENGAGEMENT_BANNER',
	'Banner2': 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
};
```
And then:
`const componentType = ComponentTypeMapping[test.channel];`